plugins/woocommerce-blocks/assets/js/base/components/cart-checkout/shipping-rates-control-package/style.scss

```
@@ -45,11 +39,19 @@
 	}
 }
 
+.wc-block-components-shipping-rates-control__package-header {
+	margin: 0 0 $gap-small 0;
```

Nitpic: We can shorten this SCSS definition like this:

```suggestion
	margin: 0 0 $gap-small;
```